core/src/sign.c

```diff
@@ -613,6 +623,14 @@ Result handle_sign_tx(InternalCommandRequest_SignTxRequest *request,
     memcpy(response->signatures[i].hash, hash, sizeof(hash));
   }
   response->signatures_count = request->inputs_count;
+  cleanup:
+  // cleanup the house be it error or success.
+  memzero((void *) seed, SHA512_DIGEST_LENGTH);
+  memzero((void *) &wallet, sizeof(HDNode));
+  for (int i = 0 ; i < MULTISIG_PARTS ; i++){
```

Comment:
nit: space before {

Reply:
I was thinking about running clang format on the whole file. thoughts ?